src/main/java/org/springframework/data/elasticsearch/client/ClientConfiguration.java

```diff
@@ -135,6 +136,14 @@ static ClientConfiguration create(InetSocketAddress socketAddress) {
 	 */
 	Duration getSocketTimeout();
 
+	/**
+	 * Returns the path prefix that should be prepended to HTTP(s) requests for Elasticsearch behind a proxy.
+	 * 
+	 * @see org.elasticsearch.client.RestClientBuilder#setPathPrefix
```

Comment:
I would not reference the Elasticsearch class here, as this `ClientConfiguration` should also be used for the reactive client, and there the `RestClientBuilder` is not used or relevant.